add-ons.html

```diff
@@ -17,6 +17,8 @@
 <p>Two different ways to integrate Solr, the Lucene based search engine, into Plone</p>
 <h3><a href="https://pypi.python.org/pypi/eea.facetednavigation">eea.facetednavigation</a></h3>
 <p>Turn folders and collections into faceted searches</p>
+<h3><a href="https://pypi.python.org/pypi/egenix-mxodbc-zopeda/">egenix-mxodbc-zopeda</a></h3>
+<p>Connect Plone to relational databases</p>
```

Comment:
Given that we have also RelStorage with a complete different scope/goal: the description could be more specific.